src/main/java/net/revelc/code/formatter/java/JavaFormatter.java

```diff
@@ -37,17 +38,37 @@
  */
 public class JavaFormatter extends AbstractCacheableFormatter implements Formatter {
 
+    @Override
```

Comment:
@delanym @ctubbsii Sorry late again here, this part should move down after the class variables and needs a bit of cleanup to match our coding style.  I can take care of later if not addressed, just noting this.  And as I think I mentioned on other PR originally, Never thought to store it this way, nice!